src/app/wallet/wallet/balances/balance.component.ts

```diff
@@ -15,10 +15,11 @@ import { takeWhile } from 'rxjs/operators';
 export class BalanceComponent implements OnInit, OnDestroy {
 
   @Input() type: string; // "total_balance", "anon_balance", "balance", "staked_balance", "blind_balance"
+  @Input() inSidebar: boolean = false; // located in sidebar?
 
   private log: any = Log.create(`balance.component ${this.type}`);
   private destroyed: boolean = false;
-
+  private availableBal: number = 0;
```

Comment:
Any particular reason this is not of the 'Amount' type?

Reply:
availableBal is the temp to store total calculation of unspent amount, That's it